styles/modern.css

```
@@ -60,7 +60,7 @@ body {
     width: 100%;
     background-color: var(--color-bg);
     color: var(--color-text-main);
-    font-family: 'Inter', system-ui, -apple-system, sans-serif;
+    font-family: 'Inter', 'Noto Sans JP', 'Segoe UI', 'Microsoft YaHei', 'Meiryo', 'Apple SD Gothic Neo', 'PingFang SC', 'Hiragino Sans', system-ui, -apple-system, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)